src/main/java/software/bernie/techarium/block/BeamBlock.java

```diff
@@ -0,0 +1,82 @@
+package software.bernie.techarium.block;
+
+import net.minecraft.block.AbstractBlock;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.material.Material;
+import net.minecraft.item.BlockItemUseContext;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.util.Direction;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.util.math.shapes.VoxelShapes;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorld;
+import org.jetbrains.annotations.Nullable;
+import software.bernie.techarium.registry.BlockRegistry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BeamBlock extends Block {
```

Comment:
How hard would it be to make this extend TechariumBlock? Ideally we preserve some consistency

Reply:
not possible due to the need of a required directionProperty which needs atlesst 2 values. This block doesn't have rotation so useless blockstates would be created